doc/user-how-to/plugin-how-to.md

```diff
@@ -0,0 +1,46 @@
+# How to install and use QGreenland Custom
+
+QGreenland Custom is a QGIS plugin for downloading a custom set of data, including data which 
+is not part of the QGreenland Core zip package. This how-to guide walks through how to install the plugin and how to use its basic functions.
+
+## Installing QGreenland Custom
+
+To get started with this guide, access the Plugin tab 
+in the top menu bar, and select 'Manage and Install Plugins'. 
+
+![menu_bar](/_images/menu_bar.png)
+
+This will open the plugin pop-up window. First, click the Settings tab and select the check box 
+labeled 'Show also experimental plugins' so that QGreenland will be included. 
+
+![plugin_settings](/_images/plugin_settings.png)
+
+Navigating to the 'All' tab in the plugin window, search for 'QGreenland Custom'. Click 'Install plugin' and close. 
+
+## Searching for data layers
+Now that you have the plugin installed, you can begin downloading and using data in QGreenland Custom.
+To access the plugin, find the QGreenland logo in the menu bar. Clicking on this icon will
+open the plugin. Initially, you should see a list of layer directories. 
+
+![plugin_search_window](/_images/plugin_search_window.png)
+
+Using the top search bar, type in a layer or keyword, such as 'towns' or 'earthquakes'. 
+Once you have located a layer of interest, click the checkbox and click next at the bottom of the window. 
+
+![selected_layer](/_images/selected_layer.png)
+
+## Downloading the data
+Next, you will need to choose a location to download the data. Select your location using the 
```

Comment:
Maybe instead of "location" we should say "storage location", "folder", or "directory"?